etc/adminhtml/system/terminal5.xml

```diff
@@ -33,6 +33,10 @@
             <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
             <config_path>payment/terminal5/isapplepay</config_path>
         </field>
+        <field id="applepaylabel" translate="label comment" type="text" sortOrder="30" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
Could you add a comment for what the label is being used for?

Reply:
Added